addons/l10n_kw/data/template/account.group-kw.csv

```diff
@@ -0,0 +1,32 @@
+"id","code_prefix_start","code_prefix_end","name","name@kw_KW"
+"kw_group_01","1000","1000","Liquidity","السيولة"
+"kw_group_02","1001","1001","Liquidity","السيولة"
```

Comment:
What is there between `1000` and `1001`?
Are the second (and third) digits useless?

Reply:
According to the PO response:
1) Bank - Prefix 1000 - Group 1000 Liquidity
2) Cash - Prefix 1009 - Group 1009 Liquidity
3) Attached accounts for Prefix 1001 - Group 1001 Liquidity
![image](https://github.com/user-attachments/assets/decf6bc1-6e4d-4b61-b041-7944da3fd234)
As we need to give bank and cash separate prefix's since it much easier for the user to create multiple accounts of each type on the fly, this was one way we decided to do it 
The generalised group is Liquidity. We are open to change it (make it more specific) once we get more feedback 
The account groups are 4 digits whereas the account codes are 6

Reply:
Strange, but ok